src/reducers/computeRenderedRows.js

```diff
@@ -159,11 +176,12 @@ function calculateRenderedRowRange(state, scrollAnchor) {
     updateRowHeight(state, rowIdx);
   }
 
+
   const { availableHeight } = scrollbarsVisibleSelector(state);
-  if (lastIndex !== undefined) {
+  if (lastIndex !== undefined || forceScrollToLastRow) {
     // Calculate offset needed to position last row at bottom of viewport
     // This should be negative and represent how far the first row needs to be offscreen
-    firstOffset = Math.min(availableHeight - totalHeight, 0);
+    firstOffset = firstOffset + Math.min(availableHeight - totalHeight, 0);
```

Comment:
It's not clear to me that this change won't affect current behavior when `lastIndex !== undefined`. Can we guarantee that `firstOffset === 0` in that case? Otherwise the offset will be different here.

Reply:
Ah ya, that's a bit confusing.  

The offset is going to always be 0 when the lastIndex is set.  That's because the last index is used to say, "scroll this row into view as the last row of the table."  Whenever we do that we're putting that row flush so it's fully in view and not offset.  You can verify this by looking at all the cases in scrollAnchor.js.

I'll add a comment here which captures something like this.

Reply:
comment added:

```
// NOTE (jordan): The first offset should always be 0 when lastIndex is defined
// since we don't currently support scrolling the last row into view with an offset.
```